.install/alpine_linux_3.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+MODE=$1 # whether to install using sudo or not
+set -e
+
+$MODE apk update
+
+$MODE apk add --no-cache build-base gcc g++ make linux-headers openblas-dev \
+    xsimd curl wget git python3 python3-dev py3-pip openssl openssl-dev \
+    tar xz which rsync bsd-compat-headers clang clang17-libclang
+
+$MODE apk add cmake --no-cache
```

Comment:
Add a comment that need to use the pakcage manager to install cmake (which is using musl)
And not call ```source install_cmake.sh $MODE``` (which is using glibc)

Reply:
Alternatively, call ```source install_cmake.sh $MODE```
And add a case for `'Alpine Linux'` installing cmake there